### PR TITLE
Put llhttp parser at dependencies list

### PR DESCRIPTION
**Pull request by juanarbol**
I saw today an error on dependencies list, `http-parser` listed on top, but `llhttp` seems to be the right name, also, `llhttp` had `http-parser` docs link.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Please check if this needs to be applied to translations.

**Comment by juanarbol** [comment]
Yeap, an update is required in:

1. ar
2. ja
3. ko
4. pt-br
5. ru
6. uk
7. zn-cn

But I don't know how to change the description in these languages; when I say description, I'm saying: "HTTP parsing is handled by a lightweight TypeScript and C library called llhttp..."

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
If anyone merges this and it's not me, please **don't squash the patches**.

File: locale/pt-br/docs/meta/topics/dependencies.md [hunk 1]
```diff
@@ -55,19 +55,19 @@ para distribuir o trabalho que não pode ser feito assíncronamente a nível de
...
 
-<!-- HTTP parsing is handled by a lightweight C library called http-parser. It is
+<!-- HTTP parsing is handled by a lightweight C library called llhttp. It is
 designed to not make any syscalls or allocations, so it has a very small
 per-request memory footprint.
 
-* [Documentation](https://github.com/joyent/http-parser/) -->
+* [Documentation](https://github.com/nodejs/llhttp) -->
 
 O parsing do protocolo HTTP é delegado a uma biblioteca leve, escrita em C,
```
Comment (juanarbol) [inline locale/pt-br/docs/meta/topics/dependencies.md:66]: C and Typescript*

I sent this PR before, and one correction/suggestion was add Typescript to description. This "error" is also in other languajes.

Comment (juanarbol) [reply locale/pt-br/docs/meta/topics/dependencies.md:66]: See: https://github.com/nodejs/nodejs.org/pull/2612#discussion_r329295703

Comment (XhmikosR) [reply locale/pt-br/docs/meta/topics/dependencies.md:66]: Yeah, well translations will need to be updated later for the Typescript part. I personally don't mind landing this as is.

Comment (juanarbol) [reply locale/pt-br/docs/meta/topics/dependencies.md:66]: Looks good to me

**Review by ghost — APPROVED** [review]
Chinese looks good to me

**Comment by ghost** [comment]
@nodejs/website , Please have a review of the changed translations for all the other languages.

**Comment by yous** [comment]
`ko/` seems okay.

**Comment by Trott** [comment]
I'm going to go ahead and land this. If any of the translations need improvements, they can certainly happen in a separate PR.

**Comment by Trott** [comment]
> If anyone merges this and it's not me, please **don't squash the patches**.

Using  'Rebase and merge" rather than "Squash and merge'".

**Comment by Trott** [comment]
People can (and still) should check the translations to make sure we didn't introduce any errors here. Thanks!

**Pull request merged**